wire-compiler/src/main/java/com/squareup/wire/WireCompiler.java

```diff
@@ -94,6 +98,7 @@
   public static final String NAMED_FILES_ONLY = "--named_files_only";
   public static final String ANDROID = "--android";
   public static final String COMPACT = "--compact";
+  public static final String CUSTOM_PROTO_ADAPTER = "--custom_proto_adapter=";
```

Comment:
hmm . . . maybe omit `custom`  and just call this `--proto_adapter` ? (All options are customizations, right?)